spk/rutorrent/src/installer.sh

```diff
@@ -98,7 +128,9 @@ postinst ()
     if [ "${APACHE_USER}" == "nobody" ]; then
         echo -e "<Directory \"${WEB_DIR}/${PACKAGE}\">\nphp_admin_value open_basedir none\n</Directory>" > /usr/syno/etc/sites-enabled-user/${PACKAGE}.conf
     else
-        echo -e "[PATH=${WEB_DIR}/${PACKAGE}]\nopen_basedir = Null" > /etc/php/conf.d/${PACKAGE_NAME}.ini
+        if [ -d "/etc/php/conf.d/" ]; then
+            echo -e "[PATH=${WEB_DIR}/${PACKAGE}]\nopen_basedir = Null" > /etc/php/conf.d/${PACKAGE_NAME}.ini
```

Comment:
This step is not needed for DSM 6.0 when using WebStation. (it's actually tackled by php worker mechanisms.